ci/collect-garbage.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash ../shell.nix
+# shellcheck shell=bash
+
+set -eux
+
+scriptroot="$(dirname "$(realpath "$0")")"
+NRHOSTS="$(ls "$scriptroot"/../morph-network/machines/*.expr.nix | wc -l)"
+
+step() {
+  host="$1"
+
+  cat <<EOF
+  - label: "nix-collect-garbage"
+    concurrency_group: ofborg-infrastructure-deploy
+    concurrency: $NRHOSTS
+    command:
+      - ./enter-env.sh morph exec --on="$host" ./morph-network/default.nix nix-collect-garbage
+    agents:
+      ofborg-infrastructure: true
+EOF
+}
+
+hosts="$(nix-instantiate -E --eval --json \
+  "builtins.attrNames
+    (builtins.removeAttrs
+      (import "$scriptroot"/../morph-network/default.nix)
+      [ \"network\" ])" \
+  | jq -r '. | to_entries | map("\(.value)") | join(" ")')"
+
+for host in $hosts; do
+  step "$host" | buildkite-agent pipeline upload
+done
```

Comment:
I think we need to print a `steps:` before each step, so maybe we want to do:

```suggestion
(
echo "steps:"
  for host in $hosts; do
    step "$host"
  done
) | buildkite-agent pipeline upload
```

Reply:
Actually, going off of https://github.com/buildkite/dynamic-pipeline-example/blob/master/.buildkite/pipeline.sh, it should only be necessary to do so once (at the beginning).

Then again, the last commit there was in 2016...